### PR TITLE
New route to course-editor

### PR DESCRIPTION
**Pull request by shiranmeir**
New route to course-editor

### CODE REVIEW

File: src/components/Screen/index.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React, { ReactElement } from 'react';
 import { Switch, Route, Redirect } from 'react-router-dom';
 
-import { COURSES_ROUTE, USERS_ROUTE } from '../../constants/routes';
+import { COURSES_ROUTE, USERS_ROUTE, COURSE_EDITOR_ROUTE } from '../../constants/routes';
 import CoursesScreen from './CoursesScreen';
 import UsersScreen from './UsersScreen';
+import EditorCourseScreen from './EditorCourseScreen';
```
Comment (dr15) [inline src/components/Screen/index.tsx:7]: Please rename `EditorCourseScreen` to `CourseEditorScreen`

File: src/constants/routes.ts [hunk 2]
```diff
@@ -23,10 +23,10 @@ export const USERS_ROUTE = {
...
 
-export const NEWCOURSE_ROUTE = {
-  id: 'new-course',
-  path: '/new-course',
-  title: 'new-course',
+export const COURSE_EDITOR_ROUTE = {
+  id: 'course-editor',
+  path: '/course-editor/new',
+  title: 'course-editor',
 };
```
Comment (dr15) [inline src/constants/routes.ts:30]: Please remove the 'new' from the path and add another route like this:
```
export const COURSE_EDITOR_ROUTE = {
  id: 'course-editor',
  path: '/course-editor',
  title: 'course-editor',
};

export const NEW_COURSE_EDITOR_ROUTE = {
  id: 'new-course',
  path: `${COURSE_EDITOR_ROUTE.path}/new`,
  title: 'new-course',
};
```

and in the `Screen/index.tsx` file make `/course-editor` redirect to `/course-editor/new`

File: src/components/Screen/EditorCourseScreen/index.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React, { ReactElement } from 'react';
+
+import WMCard from '../../common/WMCard';
+
+export default function EditorCourseScreen(): ReactElement {
```
Comment (dr15) [inline src/components/Screen/EditorCourseScreen/index.tsx:5]: Please rename `EditorCourseScreen` to `CourseEditorScreen`

File: src/components/Screen/EditorCourseScreen/index.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  return (
+    <>
+      <WMCard title="items" />
```
Comment (dr15) [inline src/components/Screen/EditorCourseScreen/index.tsx:8]: Please add a `<CourseScreenHeader ... />` element to this page

File: src/components/Screen/index.tsx (hunk 1 above)
Comment (shiranmeir) [reply src/components/Screen/index.tsx:7]: Done

File: src/constants/routes.ts (hunk 2 above)
Comment (shiranmeir) [reply src/constants/routes.ts:30]: Done

File: src/components/Screen/EditorCourseScreen/index.tsx (hunk 3 above)
Comment (shiranmeir) [reply src/components/Screen/EditorCourseScreen/index.tsx:5]: Done

File: src/components/Screen/EditorCourseScreen/index.tsx (hunk 4 above)
Comment (shiranmeir) [reply src/components/Screen/EditorCourseScreen/index.tsx:8]: Done

File: src/components/Screen/index.tsx [hunk 5]
```diff
@@ -17,6 +18,12 @@ export default function Screen(): ReactElement {
         <Route path={USERS_ROUTE.path}>
           <UsersScreen />
         </Route>
+        <Route path={NEW_COURSE_EDITOR_ROUTE.path}>
+          <EditorCourseScreen />
+        </Route>
+        <Route path="/">
```
Comment (dr15) [inline src/components/Screen/index.tsx:24]: This should be `path={COURSE_EDITOR_ROUTE.path}`
and it should redirect to `NEW_COURSE_EDITOR_ROUTE.path`

**Pull request merged**